Artesian/Artesian.SDK/Service/Query/QueryParamaters/ActualQueryParamaters.cs

```diff
@@ -29,19 +30,27 @@ public ActualQueryParamaters()
         /// <param name="transformId"></param>
         /// <param name="timezone"></param>
         /// <param name="filterId"></param>
+        /// <param name="fillerK"></param>
+        /// <param name="fillerDV"></param>
+        /// <param name="fillerP"></param>
         public ActualQueryParamaters(
             IEnumerable<int> ids, 
             ExtractionRangeSelectionConfig extractionRangeSelectionConfig, 
             ExtractionRangeType? extractionRangeType,
             string timezone,
             int? filterId,
             Granularity? granularity, 
-            int? transformId
+            int? transformId,
+            FillerKind fillerK,
```

Comment:
Please make the Filler similar to MostRecent, RangeSelection etc. So with a Type and a Config part.

Reply:
Granularity? granularity, 
int? transformId,
FillerKindType fillerK,
FillerConfig fillerConfig

config class added with default value and period but due to naming ambiguity 
will have to distinguish between  public double? FillerTimeSeriesDV { get; set; } and
public MarketAssessmentValue FillerMasDV { get; set; }. Not sure of this.

Reply:
It does make sense. Ok with that duplication, including the name of the Fields